packages/components/FocusZone/macos/RCTFocusZoneManager.m

```diff
@@ -0,0 +1,56 @@
+#import "RCTFocusZone.h"
+#import "RCTFocusZoneManager.h"
+
+@implementation RCTFocusZoneManager
+
+RCT_EXPORT_MODULE()
+
+RCT_EXPORT_VIEW_PROPERTY(disabled, BOOL)
+
+RCT_CUSTOM_VIEW_PROPERTY(focusZoneDirection, NSString, RCTFocusZone)
+{
+	if ([json isEqualToString:@"bidirectional"])
+	{
+		[view setFocusZoneDirection:FocusZoneDirectionBidirectional];
+	}
+	else if ([json isEqualToString:@"vertical"])
+	{
+		[view setFocusZoneDirection:FocusZoneDirectionVertical];
+	}
+	else if ([json isEqualToString:@"horizontal"])
+	{
+		[view setFocusZoneDirection:FocusZoneDirectionHorizontal];
+	}
+	else if ([json isEqualToString:@"none"])
+	{
+		[view setFocusZoneDirection:FocusZoneDirectionNone];
+	}
+	else
+	{
+		[view setFocusZoneDirection:[defaultView focusZoneDirection]];
+	}
+}
+
+RCT_CUSTOM_VIEW_PROPERTY(navigateAtEnd, NSString, RCTFocusZone)
+{
+	if ([json isEqualToString:@"NavigateStopAtEnds"])
+	{
+		[view setNavigateAtEnd:NavigateStopAtEnds];
+	}
+	else if ([json isEqualToString:@"NavigateWrap"] || [json isEqualToString:@"NavigateContinue"])
+	{
+		// Not yet implemented
```

Comment:
GitHub ticket? or some other tracking mechanism? should be fail somehow obviously?

Reply:
In devmain, this line was `AssertTag(false, 0x228146d7 /* tag_86u1x */);    // NYI`

Reply:
what's is the convention in furn? we can't assert or nslog. preconditionfailure might be agressive but possible.